C/primitive/elements/env.c

```diff
@@ -110,20 +110,32 @@ static void hashBuffer(sha256_midstate* result, const rawBuffer* buffer) {
 static void sha256_issuance(sha256_context* ctx, const assetIssuance* issuance) {
   switch (issuance->type) {
    case NO_ISSUANCE:
-    sha256_confAmt(ctx, &(confAmount){0});
-    sha256_confAmt(ctx, &(confAmount){0});
+    sha256_uchar(ctx, 0);
+    sha256_uchar(ctx, 0);
     break;
    case NEW_ISSUANCE:
-    sha256_confAmt(ctx, &issuance->assetAmt);
-    sha256_confAmt(ctx, &issuance->tokenAmt);
+    if (EXPLICIT == issuance->assetAmt.prefix  && 0 == issuance->assetAmt.explicit) {
```

Comment:
In 07492e02adee225020e206fb2a1f51b9f1b4c740:

Weird spacing here (and on identical lines); it also took me a while to understand what is happening here. A comment saying "serialize explicit 0s as nulls" would be helpful.

Reply:
This function soon will be entirely deleted, so I'm going to leave this as a WONTFIX.

Reply:
Sounds good to me!